### DISCUSSION TITLE
how to update the datasources dependency-track used?

### DISCUSSION THREAD

**Discussion by nancheal** [DependencyTrack/dependency-track#1084] [Q&A]
is the there any way to update the datasources for the new cves/new datasource, or the datasources will auto update background?

Regards

**Comment by stevespringett** [comment]
NVD is updated automatically. See https://docs.dependencytrack.org/datasources/nvd/

**Reply by nancheal** [reply]
thanks, the other datasources like "oss index" updated automatically too?

**Reply by stevespringett** [reply]
Last time I checked with Sonatype, OSS Index was updated daily. Dependency-Track caches responses from NPM Advisories and OSS Index for 24 hours. Constant checks for a component, for example, will result in the first check reaching out to the source and subsequent checks using the cached result until the cache expires (24 hours). When the cache is expired, DT will reach out to the sources again.

**Reply by nancheal** [reply]
a big thanks, i got it
